CRM/Core/xml/Menu/Misc.xml

```diff
@@ -85,6 +85,12 @@
   </item>
   <item>
      <path>civicrm/api</path>
+     <page_callback>CRM_Core_Page_Redirect</page_callback>
+     <page_arguments>url=civicrm/api3</page_arguments>
+     <access_arguments>access CiviCRM</access_arguments>
```

Comment:
@MikeyMJCO this adds the redirect.

Reply:
Thanks :-D missed that!

Reply:
I'll do an r-run on this shortly.

Reply:
This runs fine locally - both URLs work.